diff --git a/lib/cli/cli.js b/lib/cli/cli.js
index 5a4c6b8d53e8a01eca68b6d9d03b0f11626e77c1..2be60285c6c855417af39fb4b28b7f2928783312 100644
--- a/lib/cli/cli.js
+++ b/lib/cli/cli.js
@@ -12,7 +12,7 @@
 
 const debug = require('debug')('mocha:cli:cli');
 const symbols = require('log-symbols');
-const yargs = require('yargs/yargs');
+const yargs = require('yargs');
 const path = require('path');
 const {
   loadRc,
