diff --git a/lib/cli/cli.js b/lib/cli/cli.js
index b1b535408174dc0ec6273859e7b0ae3e6b9e0962..1ccee25f2822c306a37db28f55342ced5d21bc51 100644
--- a/lib/cli/cli.js
+++ b/lib/cli/cli.js
@@ -12,7 +12,7 @@
 
 const debug = require('debug')('mocha:cli:cli');
 const symbols = require('log-symbols');
-const yargs = require('yargs/yargs');
+const yargs = require('yargs');
 const path = require('path');
 const {
   loadRc,
